.github/workflows/cherry_pick_comment.yml

```diff
@@ -0,0 +1,19 @@
+name: Cherry Pick On Comment
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  cherry-pick:
+    name: Cherry Pick
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/cherry-pick')
```

Comment:
iiuc this will allow anyone adding the comment to trigger cherry-pick right?
saw in other places this check added as well:
```
github.event.comment.author_association == 'CONTRIBUTOR'
```

so we need to adapt something like this here

Reply:
Correct. This is a good idea.